tests/foreman/api/test_discovery.py

```diff
@@ -1,9 +1,38 @@
 # -*- encoding: utf-8 -*-
 """API Tests for foreman discovery feature"""
-from robottelo.common.decorators import stubbed
+from ddt import ddt
+from fauxfactory import gen_string, gen_ipaddr, gen_mac
+from nailgun import entities
+from robottelo.common.decorators import stubbed, data, run_only_on
 from robottelo.test import APITestCase
 
 
+def _create_discovered_host(name=None, ipaddress=None, discovery_bootif=None):
+    """Creates discovered host by uploading few fake facts.
+
+    :param str name: Name of discovered host. If ``None`` then a random
+        value will be generated.
+    :param str ipaddress: A valid ip address.
+        If ``None`` then then a random value will be generated.
+    :return: A ``DiscoveredHost`` object.
```

Comment:
The server's response (likely a dict) is returned. A `DiscoveredHost` object is not returned.